05-assistive-chatbot/chatbot/llms/anthopic_client.py

```diff
@@ -0,0 +1,31 @@
+from anthopic import Anthropic
+
+CLIENT_NAME = "anthropic"
+# These names will be associated with this Python module
+MODEL_NAMES = ["claude-3-opus-20240229"]
+
+
+def init_client(model_name, settings):
+    return OpenaiLlmClient(model_name, settings)
```

Comment:
Would prefer this be called AnthropicClient

Reply:
Oops! Copy-paste typo. Nice catch!